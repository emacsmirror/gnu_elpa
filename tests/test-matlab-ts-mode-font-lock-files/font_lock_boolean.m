% -*- matlab-ts -*-

a = true;
b = false;

if true || false
end
