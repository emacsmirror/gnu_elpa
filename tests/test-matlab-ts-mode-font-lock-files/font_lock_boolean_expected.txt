c ccc ccccccccc ccc

v o BBBBD
v o BBBBBD

kk BBBB oo BBBBB
kkk
